core/runtime/ops.rs

```diff
@@ -698,7 +726,7 @@ mod tests {
 
   /// Tests v8 types without a handle scope
   #[allow(clippy::needless_lifetimes)]
-  #[op2(core)]
+  #[op2(core, fast)]
```

Comment:
Did compiler now force you to mark it as fast :D ?

Reply:
It did : 😂